tetris.py

```diff
@@ -229,6 +230,7 @@ def resize_to_fit(self):
             self.current_block_center = (2, 0)
         else:
             self.current_block_center = (1, 1)
+        self.draw_block()
```

Comment:
This is confusing. I really wouldn't expect a function named `resize_to_fit` to draw anything to the canvas.